integration-test/1491-generic-network-fixes.py

```diff
@@ -0,0 +1,4231 @@
+                'name': u'Mahendra Rajmarg',
+                'name:en': u'Mahendra Highway',
+                'name:ne': u'महेन्द्र राज्मार्ग्',
+                'name:pl': u'Mahendra',
+                'network': u'np:national',
+                'ref': u'H01',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q3595565',
+                'wikipedia': u'en:Mahendra Highway',
+            }, ways=[27030698]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 27030698,
+                'network': u'NP:national',
+            })
+
+    def test_npregional(self):
+        import dsl
+
+        z, x, y = (16, 48796, 27744)
+
+        self.generate_fixtures(
+            dsl.is_in('NP', z, x, y),
+            # https://www.openstreetmap.org/way/352615958
+            dsl.way(352615958, dsl.tile_diagonal(z, x, y), {
+                'highway': u'primary',
+                'name': u'Birtamod — Chandragadhi',
+                'ref': u'F1',
+                'source': u'openstreetmap.org',
+                'surface': u'paved',
+            }),
+            dsl.relation(1, {
+                'distance': u'12.53 km',
+                'name': u'Birtamod — Chandragadhi',
+                'network': u'np:regional',
+                'ref': u'F1',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[352615958]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 352615958,
+                'network': u'NP:regional',
+            })
+
+    def test_phnational(self):
+        import dsl
+
+        z, x, y = (16, 55524, 30719)
+
+        self.generate_fixtures(
+            dsl.is_in('PH', z, x, y),
+            # https://www.openstreetmap.org/way/460910438
+            dsl.way(460910438, dsl.tile_diagonal(z, x, y), {
+                'highway': u'trunk',
+                'int_ref': u'AH26',
+                'lanes': u'2',
+                'name': u'Maharlika Highway',
+                'nat_name': u'Daang Maharlika (LT)',
+                'official_name': u'Daang Maharlika Highway',
+                'ref': u'1;AH26',
+                'source': u'openstreetmap.org',
+                'surface': u'concrete',
+            }),
+            dsl.relation(1, {
+                'name': u'N1',
+                'network': u'ph:nhn',
+                'nhn:class': u'primary',
+                'ref': u'1',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[460910438]),
+            dsl.relation(2, {
+                'int_ref': u'AH26',
+                'int_ref:colour': u'blue',
+                'name': u'National Highway',
+                'name:en': u'National Highway',
+       
```

Comment:
Please test shield_text of `D555` (remove space)?

Reply:
I also saw evidence of `TR:DY` network during research last year – what's the right network value here?

Reply:
`DY` would seem to be an abbreviation of [Devlet yolu](https://en.wikipedia.org/wiki/Turkish_State_Highway_System), which means "State road". However, I can't find any relations with that `network` value at the moment. The ones I see are:

* `TR/35-road`
* `TR-motorway`
* `TR-road`
* `TR-roads`

However, most roads aren't part of any relation. But all the information we need seems to be in the `ref` anyway. Perhaps the `TR:DY` ones got fixed up in some way since last year?

Follow up in #1517.